x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition.ts

```diff
@@ -34,7 +33,7 @@ export const evaluateCondition = async ({
   compositeSize,
   filterQuery,
   lookbackSize,
-  startTime,
+  executionTimestamp,
```

Comment:
Thanks for choosing a descriptive name :+1: Can we use the same name over in #128019? :innocent:

Reply:
Yeah... I will update #128019 to reflect some of the feedback in this PR.